content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -373,3 +374,175 @@ Alternatively, the annotation can be added to a namespace, which will result in
 all services in that namespace to opt-in to automatic instrumentation. See the
 [Operators auto-instrumentation documentation](https://github.com/open-telemetry/opentelemetry-operator/blob/main/README.md#opentelemetry-auto-instrumentation-injection)
 for more details.
+
+## Troubleshooting
+
+If you run into problems trying to auto-instrument your code, here are a few
+things that you can try.
+
+### **1- Did the Instrumentation resource install?**
```

Comment:
- Headings are already in bold, so we don't need the extra bolding markdown syntax.
- I'd suggest dropping the numbers too.

```suggestion
### Did the Instrumentation resource install?
```